### PR TITLE
ko-KR: Fix typo

### PR DESCRIPTION
**Pull request by ParkSB**
I fixed a grammatical error.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Tagging @telk5093 . (You can discuss in Korean, if necessary.)

You accidentally replaced a regular space with a CJK space. This will need to be reverted.

**Comment by telk5093** [comment]
What do you mean about regular space? I couldn't find a different type of space in his commit diff.
(There is no such 'CJK space' in korean. The space that is used in korea is the same one with yours, ```\20```)
And ```늘일``` was my mistake, it must be ```늘릴`` as he intended to.

@ParkSB 혹시 STR_3214 뒷 부분에 공백 부분 건드리셨나요, 제가 보기엔 건드린 거 없어보이는데 아니라면 바로 merge해달라고 하면 될 것 같네요.

**Comment by telk5093** [comment]
What do you mean about regular space? I couldn't find a different type of space in his commit diff.
(There is no such 'CJK space' in korean. The space that is used in korea is the same one with yours, ```\20```)

@ParkSB 혹시 STR_3214 뒷 부분에 공백 부분 건드리셨나요, 제가 보기엔 건드린 거 없어보이는데..

**Comment by ParkSB** [comment]
@Gymnasiast As you can see from the commit diff, I haven't changed any space. I changed just one letter.
@telk5093  네, 공백은 안 건드리고 `늘일`을 `늘릴`로 바꾸기만 했습니다.

**Comment by telk5093** [comment]
@Gymnasiast I think you may merge this, there seems to be no problem.

File: data/language/ko-KR.txt [hunk 1]
```diff
@@ -3222,7 +3222,7 @@ STR_3210    :다음 단계로:
 STR_3211    :지도 크기:
 STR_3212    :{POP16}{COMMA16} x {PUSH16}{COMMA16}
 STR_3213    :더 이상 맵 크기를 줄일 수 없습니다
-STR_3214    :더 이상 맵 크기를 늘일 수 없습니다
+STR_3214    :더 이상 맵 크기를 늘릴 수 없습니다
```
Comment (Gymnasiast) [inline data/language/ko-KR.txt]: The space between `STR_3214` and `:` is what I'm talking about. Our system cannot handle this.

Comment (ParkSB) [reply data/language/ko-KR.txt]: Thanks for the warning. I will send a new PR after fix this issue. Should I close #1395?

**Pull request closed without merging**